config.json

```diff
@@ -54,6 +54,21 @@
             "required": false,
             "type": "textarea",
             "repeatable": false
+        },
+        {
+            "key": "use_goprod",
+            "name": "Use goprod module?",
+            "type": "radio",
```

Comment:
Interesting note: making this a checkbox with only the "Yes" option did not actually allow toggling